fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.json

```diff
@@ -265,14 +266,20 @@
    "fieldtype": "Data",
    "label": "Gender",
    "read_only": 1
+  },
+  {
+   "default": "0",
+   "fieldname": "is_anonymous",
+   "fieldtype": "Check",
+   "label": "Is Anonymous?"
```

Comment:
Anonymous means something else. "Is Private" is the right word here, only they can see it.